src/lib/components/admin/Functions/FunctionEditor.svelte

```
@@ -1,8 +1,7 @@
 <script>
-	import { getContext, createEventDispatcher, onMount, tick } from 'svelte';
+	import { getContext, onMount, tick } from 'svelte';
```

ESLint

[error] 'onMount' is defined but never used.

(@typescript-eslint/no-unused-vars)